core/src/main/java/nl/nn/adapterframework/util/AppConstants.java

```diff
@@ -297,30 +316,55 @@ private synchronized void load(ClassLoader classLoader, String filename, String
 	}
 
 	/**
-	 * Add property which can be used to substitute variables in the value of
-	 * the property ADDITIONAL.PROPERTIES.FILE.
+	 * Add property only in the local AppConstants!
+	 * Try to avoid using this method and use {@link #setProperty(String, String)} if you want to set the property globally!
+	 * 
+	 * This method is used by {@link Properties#load(InputStream)} to add all properties found (in a file/stream) 
+	 * to the {@link Hashtable}.
+	 * @Deprecated Use {@link #setProperty(String, String)} instead!
+	 */
+	@Deprecated
+	@Override
+	public synchronized Object put(Object key, Object value) {
+		return super.put(key, value);
+	}
+
+	/**
+	 * Add property to global AppConstants
 	 */
-	public void putAdditionalPropertiesFilesSubstVarsProperty(String name, String value) {
```

Comment:
Snappen we waar deze method voor bedoeld was? Hij lijkt nergens gebruikt te worden.

Reply:
Hij werd gebruikt voor de application.server.type property, maar die heb ik in een eerdere PR (#297) al obsolete gemaakt

Reply:
Eigenlijk was het een 'put before load', die alleen gebruikt werd in het rare geval als je tomcat6 had opgegeven ipv tomcat (of niets voor auto-detect)